verifier/src/test/java/com/aws/jverify/ResolutionErrorsDoubleOperators.java

```diff
@@ -0,0 +1,73 @@
+package com.aws.jverify;
+
+@SuppressWarnings("ConstantValue")
+class ResolutionErrorsDoubleOperators {
+    public void foo() {
+        var l = 3.0;
+        var r = 3.0;
+        
+        var equals = l == r;
+//                     ^ error: operator ==(double,double) is not supported
+        var notEquals = l != r;
+//                        ^ error: operator !=(double,double) is not supported
+        
+        l++;
+//       ^ error: operator ++(double) is not supported
+        //check(l == 4f);
+        l--;
+//       ^ error: operator --(double) is not supported
+        //check(l == 3f);
+        ++l;
+//      ^ error: operator ++(double) is not supported
+        //check(l == 4f);
+        --l;
+//      ^ error: operator --(double) is not supported
+        //check(l == 3f);
+        
+        var multiplication = l * r;
+//                             ^ error: operator *(double,double) is not supported
+        //check(multiplication == 9f);
+        var division = l / r;
+//                       ^ error: operator /(double,double) is not supported
+        //check(division == 1f);
+        var remainder = l % 2f;
```

Comment:
nit just for consistency:
```suggestion
        var remainder = l % 2d;
```